src/main/jbake/content/community/docs/developer-guide/alerts-v2.adoc

```diff
@@ -872,6 +872,253 @@ Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/htt
+    {...},
+    {...}
+  ],
+  "groupLabels":{},
+  "commonAnnotations":{},
+  "externalUrl":"...",
+  "version":"...",
+  "groupKey":"..."
+}
+```
+
+A Prometheus Notification is mapped into a Hawkular Event with the following convention: +
+ +
+
+[cols="15,15", options="header"]
+|=======================
+|Prometheus Notification|Hawkular Event
+
+|notification.groupLabels.tenant|event.tenantId
+|notification.groupLabels.dataId|event.dataId
+|"prometheus"|event.category
+|notification.commonAnnotations.description|event.text
+|notification.commonAnnotations.summary|event.text
+|notification.groupLabels.alertName|event.text
+|notification.groupLabels.alertName|event.context.alertName
+|notification|event.context.json
+
+|=======================
+
+
+ notification.groupLabels.dataId -> event.dataId
+
+
+*Body*
+
+[cols="15,35,^15", options="header"]
+|=======================
+|Required|Description|Data Type
+
+|Yes|Prometheus Notification|Object
+
+|=======================
+
+
+
+*Response*
+
+*Status codes*
+[cols="^20,55,^25", options="header"]
+|=======================
+|Status Code|Reason|Response Model
+
+|200|Successfully processed|Success
+|400|Bad Request/Invalid Parameters|<<ApiError>>
+
+|=======================
+
+
+
+==============================================
+
+== Apache Kafka Integration
+
+=== Apache Kakfa Alerter
+
+The Kafka Alerter will listen for triggers tagged with "Kafka" tag. The Alerter will create a consumer to a Kafka Topic
+with the config provider into the Trigger context. The Alerter will convert Kafka records into Hawkular Data or Events
+and send them into the Alerting engine.
+
+The Kafka Alerter uses the following conventions for trigger tags and context:
+
+[cols="^2,10"]
+|=======================
+| *Required* |
+_trigger.tags["Kafka"] = "<value reserved for future uses>"_ +
+ +
+ An "Kafka" tag is required for the alerter to detect this trigger will listen to a Kafka topic. +
+ Value is not necessary, it can be used as a description, it is reserved for future uses. +
+  +
+ i.e. +
+ trigger.tags["Kafka"] = ""                          // Empty value is valid +
+ trigger.tags["Kafka"] = "OpenShift Kafka System"    // It can be used as description
+| *Required* |
+_trigger.context["kafka.*"] = "<kafka native properties>"_ +
+ +
+ Kafka Consumer properties are prefixed with "kafka." in the Trigger context. +
+ kafka.key.deserializer and kafka.value.deserializer point to StringDeserializer as default if not present. +
+  +
+i.e. +
+trigger.context["kafka.bootstrap.servers"] = "localhost:9092" +
+trigger.context["kafka.group.id"] = "kafka-trigger-group" +
+ +
+See the link:https://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html[KafkaConsumer reference] for more info +
+| *Required* |
+_trigger.context["topic"] = "<kafka topic to listen>"_ +
+ +
+ It defines the Kafka Topic a trigger will listen. In the current version only a single topic for Trigger is allowed. +
```

Comment:
"It defines the Kafka Topic on which the trigger will listen.  The current version allows only a single topic per Trigger."